pkg/skaffold/deploy/kubectl.go

```diff
@@ -46,6 +46,7 @@ type KubectlDeployer struct {
 	workingDir         string
 	kubectl            deploy.CLI
 	insecureRegistries map[string]bool
+	debugConfig        latest.DebugConfig
```

Comment:
We usually try to avoid storing and passing around structs that can contain a lot of values. Maybe just store the repo location? (I'm not sure about that)